bodhi/server/scripts/approve_testing.py

```diff
@@ -103,6 +103,12 @@ def main(argv=sys.argv):
                 if update.autotime and update.days_in_testing >= update.stable_days:
                     print(f"Automatically marking {update.alias} as stable")
                     update.set_request(db=db, action=UpdateRequest.stable, username="bodhi")
+                    # For updates that do not stay in testing at all, mark the update as stable
+                    # Do note that if we ever configure a release which is composed by bodhi with
+                    # mandatory_days_in_testing = 0, its updates will end up being marked as stable
+                    # before the compose. This may lead to some unexpected behavior.
+                    if update.release.mandatory_days_in_testing == 0:
```

Comment:
```suggestion
                    if not update.release.composed_by_bodhi:
```

Turns out we have this handy boolean flag on the release object which was added ([here](https://github.com/fedora-infra/bodhi/issues/2317)), we should use it here.

Reply:
Oh nice this should alleviate most of the concerns the comment speaks about, cool!